### ISSUE TITLE
Keys starting with dot lose it when exported to JSON

### ISSUE THREAD

**Issue by ss-gxp** [WeblateOrg/weblate#4560]
Version: _4.2.2 docker image_
Аffected: _UI and API_

For example key `.0001` turns into:
```json
{
    "0001": "..."
}
```
As a result, loading back is not possible - keys not found.

**Comment by ss-gxp** [comment]
Adding.
Did a little research on the situation.
This is relevant for nested JSON when the top level has a key as an empty string.

```json
{
    "": {
        "0001": "..."
    }
}
```

After import from the repository, the key gets the value `.0001`.

When exporting, for example, to csv, the key retains the correct value.
Exporting to json dot separating the levels disappears.

**Comment by nijel** [comment]
This should be fixed with translate-toolkit 3.1.0. See #2149 and https://github.com/translate/translate/pull/4091 for more details.

It will be available in the Docker image soon, see https://github.com/WeblateOrg/docker/pull/831
